src/main/java/TaskException.java

```diff
@@ -0,0 +1,5 @@
+public class TaskException extends Exception {
```

Comment:
Maybe have this extend LukeException ? e.g. LukeTaskException. Won't have to check for LukeException | TaskException for each relevant exception.